src/OpenTelemetry.Resources.Host/HostDetector.cs

```diff
@@ -95,6 +98,37 @@ public Resource Detect()
                 attributes.Add(new(HostSemanticConventions.AttributeHostId, machineId));
             }
 
+            try
+            {
+                var hostEntry = Dns.GetHostEntry(Environment.MachineName);
+                var ips = hostEntry.AddressList.Where(x => !IPAddress.IsLoopback(x))
+                    .Select(x => x.ToString())
+                    .Distinct()
+                    .Where(x => !string.IsNullOrEmpty(x))
+                    .ToArray();
+                attributes.Add(new(HostSemanticConventions.AttributeHostIp, ips));
```

Comment:
According [to the documentation](https://opentelemetry.io/docs/specs/semconv/resource/host/) these are both opt-in, so shouldn't be collected by default.